docs/Reference/API-Methods.md

```diff
@@ -4256,50 +4490,62 @@ per transaction in the order the transactions were executed.
     }
     ```
 
-## EEA Methods
+## EEA methods
 
 !!! note
-    The `EEA` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api)
-    or [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options to enable the `EEA` API methods.
+
+    The `EEA` API methods are not enabled by default for JSON-RPC. To enable the `EEA` API methods,
+    use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
+    [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
 ### eea_sendRawTransaction
 
-Distributes the [private transaction](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md),
-generates the [Privacy Marker Transaction](../Concepts/Privacy/Private-Transaction-Processing.md) and submits it to the transaction pool,
-and returns the transaction hash of the [Privacy Marker Transaction](../Concepts/Privacy/Private-Transaction-Processing.md).
+Distributes the
+[private transaction](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md),
+generates the [Privacy Marker Transaction](../Concepts/Privacy/Private-Transaction-Processing.md)
+and submits it to the transaction pool, and returns the transaction hash of the
+[Privacy Marker Transaction](../Concepts/Privacy/Private-Transaction-Processing.md).
 
-The signed transaction passed as an input parameter includes the `privateFrom`, [`privateFor` or `privacyGroupId`](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md#eea-compliant-or-besu-extended-privacy),
+The signed transaction passed as an input parameter includes the `privateFrom`,
+[`privateFor` or `privacyGroupId`](../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md#eea-compliant-or-besu-extended-privacy),
 and `restriction` fields.
 
-To avoid exposing your private key, create signed transactions offline and send the signed transaction
-data using `eea_sendRawTransaction`.
+To avoid exposing your private key, create signed transactions offline and send the signed
+transaction data using `eea_sendRawTransaction`.
 
 !!! important
-    For production systems requiring private transactions, we recommend using a network
-    with a consensus mechanism supporting transaction finality. For example, [IBFT 2.0](../HowTo/Configure/Consensus-Protocols/IBFT.md).
-    
-    Using private transactions with [pruning](../Concepts/Pruning.md) or [fast sync](CLI/CLI-Syntax.md#sync-mode)
-    is not supported.
 
-    Besu does not implement [`eea_sendTransaction`](../HowTo/Send-Transactions/Account-Management.md).
+    For production systems requiring private transactions, we recommend using a network with a
```

Comment:
According to https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we we should try another way write this sentence as here it doesn't explain the benefits of doing so.
So perhaps "For production systems requiring private transactions, use a network with a consensus mechanism supporting transaction finality to make sure the private state can not get inconsistant with the chain. For example, IBFT2 provides the required finality."

Reply:
Reworded slightly, but updated accordingly.